.github/workflows/auto-release.yaml

```
@@ -126,18 +142,25 @@ jobs:
   # generate changelog and update to github releases pages
   release-notes:
     needs: [release-chart,release-image]
+    # updates the github release page
+    permissions:
+      contents: write
     uses: ./.github/workflows/call-release-notes.yaml
     with:
       ref: ${{ needs.ensure-tag.outputs.tag }}
```

actionlint (1.7.12)

[error] property "ensure-tag" is not defined in object type {release-chart: {outputs: {artifact: string}; result: string}; release-image: {outputs: {}; result: string}}

(expression)